lib/Runtime/Library/JavascriptSymbol.cpp

```diff
@@ -251,8 +257,7 @@ namespace Js
             *value = left->GetValue() == JavascriptSymbolObject::FromVar(right)->GetValue();
             break;
         default:
-            *value = JavascriptOperators::Equal_Full(right, left, requestContext);
-            break;
+            return FALSE;
```

Comment:
Doesn't this function return the equality via the value parameter? We're not setting *value = false here. I think returning FALSE has the effect of marking the call to RecyclableObject::Equals as a failure. Not sure if that will have an impact but I think we should set value to false and return TRUE in this case. The default branch would be empty, then.